packages/bluebird/src/semantic-ui-react/_dropdown.scss

```diff
@@ -0,0 +1,1513 @@
+//       Selection
+//   ---------------*/
+
+// /* Displays like a select box */
+.ui.selection.dropdown {
+  //   cursor: pointer;
+  //   word-wrap: break-word;
+  line-height: $line-height-small;
+  //   white-space: normal;
+  //   outline: 0;
+  //   -webkit-transform: rotateZ(0deg);
+  //   transform: rotateZ(0deg);
+  //   min-width: 14em;
+  min-height: 40px; // need variable
+  background: $white;
+  //   display: inline-block;
+  padding: ($padding-base-vertical - 1)
+    ($padding-large-horizontal * 2)
+    ($padding-base-vertical - 1) $padding-base-horizontal;
+  color: $text-color;
+  //   -webkit-box-shadow: none;
+  //   box-shadow: none;
+  border: 1px solid $input-border;
+  border-radius: $border-radius-small;
+  //   -webkit-transition: width 0.1s ease, -webkit-box-shadow 0.1s ease;
+  //   transition: width 0.1s ease, -webkit-box-shadow 0.1s ease;
+  //   transition: box-shadow 0.1s ease, width 0.1s ease;
+  //   transition: box-shadow 0.1s ease, width 0.1s ease,
+  //   -webkit-box-shadow 0.1s ease;
+}
+
+// .ui.selection.dropdown.visible,
+// .ui.selection.dropdown.active {
+//   z-index: 10;
+// }
+
+// select.ui.dropdown {
+//   height: 38px;
+//   padding: 0.5em;
+//   border: 1px solid rgba(34, 36, 38, 0.15);
+//   visibility: visible;
+// }
+
+.ui.selection.dropdown > .search.icon,
+.ui.selection.dropdown > .delete.icon,
+.ui.selection.dropdown > .dropdown.icon {
+  //   cursor: pointer;
+  //   position: absolute;
+  //   width: auto;
+  //   height: auto;
+  font-size: $font-size-large;
+  line-height: $line-height-1;
+  //   top: 0.78571429em;
+  right: $padding-large-horizontal;
+  //   z-index: 3;
+  margin: -$padding-large-vertical;
+  padding: ($padding-small-vertical - 1) $padding-small-horizontal $padding-base-vertical;
+  //   opacity: 0.8;
+  //   -webkit-transition: opacity 0.1s ease;
+  //   transition: opacity 0.1s ease;
+}
+
+// /* Compact */
+// .ui.compact.selection.dropdown {
+//   min-width: 0px;
+// }
+
+// /*  Selection Menu */
+.ui.selection.dropdown .menu {
+  //   overflow-x: hidden;
+  //   overflow-y: auto;
+  //   -webkit-backface-visibility: hidden;
+  //   backface-visibility: hidden;
+  //   -webkit-overflow-scrolling: touch;
+  //   border-top-width: 0px !important;
+  //   width: auto;
+  //   outline: none;
+  margin: $padding-small-vertical -1px;
+  padding: $padding-base-vertical 0;
+  //   min-width: calc(100% + 2px);
+  //   width: calc(100% + 2px);
+  border-radius: $input-border-radius;
+  -webkit-box-shadow: $shadow-depth-3;
+  box-shadow: $shadow-depth-3;
+  //   -webkit-transition: opacity 0.1s ease;
+  //   transition: opacity 0.1s ease;
+}
+
+// .ui.selection.dropdown .menu:after,
+// .ui.selection.dropdown .menu:before {
+//   display: none;
+// }
+
+// /*--------------
+//       Message
+//   ---------------*/
+
+.ui.selection.dropdown .menu > .message {
+  padding: $padding-base-vertical $padding-base-horizontal;
+}
+@media only screen and (max-width: 767px) {
```

Comment:
Media queries should come from screen vars.

Reply:
Fixing this.